### PR TITLE
Added event endpoint

### PR DESCRIPTION
**Pull request by selmagudmundsen**
### PR Type 🍏
What kind of change does this PR introduce?
```
[ ] Bugfix
[x] Feature
[x] Database
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] Documentation content changes
[ ] Tests
[ ] Other
```
[Kanban Link](https://www.notion.so/bekks/742fa46fd71e47728aff33e5a32e125e?v=4fe4766ee0f54d348a595b3864658662&p=56be80052d7247b8a2f16d31f2c7e8d3&pm=s)

### Context 🤷‍♀️
Have to add an endpoint in. backend in order to save events that are fetched from the journey API in frontend. 
### What's new? 👶
Added a addEvent function and functionality so all events are set to false when a new event is created.

### Screenshots 🖼️
Add pictures if applicable.

### CODE REVIEW

File: src/main/kotlin/org/entur/norgesturbackend/norgestur/repository/EventRepository.kt [hunk 1]
```diff
@@ -20,4 +22,10 @@ interface EventRepository : JpaRepository<Event, Int> {
     fun findEventByEventName(
         @Param("eventName") eventName: String
     ): Event?
+
+
+    @Modifying
+    @Transactional
+    @Query("UPDATE Event e SET e.isActive=false")
```
Comment (oscarahalvorsen) [inline src/main/kotlin/org/entur/norgesturbackend/norgestur/repository/EventRepository.kt:29]: Hadde kalt renamet e til event, fordi ikke så god praksis med variabler som man ikke vet hva betyr, selv om det her er en liten query

**Review by oscarahalvorsen — CHANGES_REQUESTED** [review]
Veldig bra jobb!👍
Kun en liten kommentar, selv om det kanskje er litt pirk

File: src/main/kotlin/org/entur/norgesturbackend/norgestur/repository/EventRepository.kt (hunk 1 above)
Comment (selmagudmundsen) [reply src/main/kotlin/org/entur/norgesturbackend/norgestur/repository/EventRepository.kt]: fixed

**Review by oscarahalvorsen — APPROVED** [review]
Herlig jobbet! Dette er bra

**Pull request merged**